### ISSUE TITLE
Doubling a class with a static function causes error.

### ISSUE THREAD

**Issue by bitwes** [bitwes/Gut#127]
Doubling a class with a static function results in:
`Function signature doesn't match the parent. Parent signature is: ...`

The error text doesn't explicitly state that "static" is missing but I'm pretty sure that's the cause of the error.

**Comment by bitwes** [comment]
For reference, here is the error:
```
Debugger Break, Reason: 'Parser Error: Function signature doesn't match the parent. Parent signature is: 'Variant this_is_a_static_method()'.'
```

**Comment by bitwes** [comment]
I can't find a way from the info in `get_method_list` to determine if a function is static or not.  The quick fix would be to allow the user to specify a list of static methods.

This also brings up the question of how they should be stubbed, if at all.  I'm not sure I can stub static methods, they may just have to be removed from the doubled code.

**Comment by bitwes** [comment]
Created a proposal in Godot, upvoting the issue there could make things move along faster:  https://github.com/godotengine/godot-proposals/issues/152

**Comment by bitwes** [comment]
Added method `ignore_method_when_doubling`.  You must call this method with every static method before doubling your script in order for doubling with static methods to work.  Methods added to the ignore list cannot be stubbed or spied on.  The ignore list is cleared after each test.

**Comment by bitwes** [comment]
Implemented in 6.8.1
